src/ontology/mondo-ingest.Makefile

```diff
@@ -275,3 +274,30 @@ mappings/mondo-sources-all-lexical.sssom.tsv: $(SCRIPTSDIR)/match-mondo-sources-
 	# The $^ includes the python script, basically all paramters after the colon :
 
 lexical_matches: mappings/mondo-sources-all-lexical.sssom.tsv
+
+#############################
+###### Slurp pipeline #######
+#############################
+slurp/:
+	mkdir -p $@
+
+# TODO: Move this to Makefile. how, when I can't edit it here?
```

Comment:
@matentzn: `component-download-mondo.owl`: Need to move this to Makefile. What is standard way for me to do that? I know I'm not supposed to edit it in `mondo-ingest` directly.

Reply:
You mentioned that you will probably refactor away `component-download-mondo.owl`. It might be good for me to know at some point how to make updates that will affect `src/ontology/Makefile` for future cases though.

Reply:
This is where you do it:

https://github.com/INCATools/ontology-development-kit/blob/master/template/src/ontology/Makefile.jinja2

But I would suggest you make issues before you make PRs, just to save you time for PR revisions.

Reply:
Done and done!
- Issue: https://github.com/INCATools/ontology-development-kit/issues/677
- PR: https://github.com/INCATools/ontology-development-kit/pull/676

Reply:
Sorry, my answer was generic: if you want to _ever_ change the `Makefile` logic, that is the file you need to make a pull request on. 

for now, keep this goals in mondo-ingest.Makefile. If we wanted to add them to Makefile, we would have to add `Mondo` as a source to the `mondo-ingest`, which may make some sense.

Reply:
OK. I think we don't need to bring this up at the meeting. Everything makes sense now.

FYI we had another, second Mondo component related target that was commented out. I think we don't need it anymore, so I'm gonna delete:
```
.PHONY: component-download-mondo.owl
component-download-mondo.owl: | $(TMPDIR)
	if [ $(MIR) = true ] && [ $(COMP) = true ]; then $(ROBOT) merge -I http://purl.obolibrary.org/obo/mondo.owl \
	annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) -o $(TMPDIR)/$@.owl; fi

# TODO: (a) Move this to Makefile, or (b) refactor this away, or (c) DELETE this goal.
# ...I think (c) is most likely, as `reports/mirror_signature-%.tsv` needs `component-download-%.owl`.
# ...`reports/component_signature-%.tsv` is what needs `components/%.owl`, but that shouldn't be needed here.
# $(COMPONENTSDIR)/mondo.owl: component-download-mondo.owl
# 	if [ $(COMP) = true ] ; then if cmp -s $(TMPDIR)/component-download-mondo.owl.owl $@ ; then echo "Component identical."; else echo "Component is different, updating." && cp $(TMPDIR)/component-download-mondo.owl.owl $@; fi; fi

Reply:
Probably was fine to delete `$(COMPONENTSDIR)/mondo.owl:`. Gonna mark this resolved.